starlette_core/rubbish.py

```diff
@@ -0,0 +1,4 @@
+import datetime
+
+def bad_input_variable(today=datetime.datetime.now()):
```

Comment:
This would always return the date based on when the file is first pulled in and not today.

A better example:
```
def bad_input_variable(today=None):
    return today or datetime.datetime.now()
```